src/main-app.js

```diff
@@ -140,11 +159,47 @@ export class MainApplication extends LitElement {
       bottom: 16px;
       right: 16px;
     }
+
+    .btn-scan {
+      fill: #018786;
+    }
+
+    .disabled-scan {
```

Comment:
mwc-icon-button should already take care of these... or there is a toggle button.

https://mwc-demos.glitch.me/demos/icon-button-toggle.html